luigi/contrib/spark.py

```diff
@@ -122,6 +122,10 @@ def jars(self):
         return self._list_config(configuration.get_config().get("spark", "jars", None))
 
     @property
+    def packages(self):
+        return self._list_config(configuration.get_config().get("spark", "packages", None))
```

Comment:
Yes, see http://luigi.readthedocs.org/en/latest/api/luigi.configuration.html. As an advantage, you get command line parameters for free.